game/game/movealgo.cpp

```diff
@@ -708,6 +718,10 @@ void MoveAlgo::startDive() {
     }
   }
 
+void MoveAlgo::applyAirPhysics() {
+  setInAir(true);
```

Comment:
Assuming that condition `isInAir == true && isSwim == true` can be caused only by this call. Is this an complicated way to call `setAsSwim (false);` ?